ports/cpp/cmake/CompileOptions.cmake

```diff
@@ -1,4 +1,4 @@
-set(CMAKE_CXX_STANDARD 23)
+set(CMAKE_CXX_STANDARD 20)
```

Comment:
Downgraded, because `GCC 11` distributed with `ubuntu-latest` does not support `C++23`, so building this library on Ubuntu computers could be problematic.